apps/user-profile/src/components/modules/LaunchYourJourney/LaunchYourJourney.js

```diff
@@ -0,0 +1,98 @@
+    };
+
+    const IdeaSpacePage = (e) => {
+    router.push('/ideaspace')
+    };
+
+    const LearnPage = (e) => {
+    router.push('/learn')
+    };
+
+    const JoinPage = (e) => {
+    router.push('/join')
+    };
+
+    return(
+        <>
+
+        <PageContainer>
+            <HeaderSection style={{ backgroundImage:`url(${headerBG})` }}>
+                <Header>
+                    <Typography style={{color: "white", fontSize: "80px"}}>Launch Your Journey!</Typography>
+                </Header>
+                <SubHeader>
+                    <Typography type="pLarge" style={{color: "white"}}>
+                        Dev Launchers is here to launch your tech career into the stratosphere. Our community is full of cosmic adventurers, all ready to help you reach new dimensions of success.
+                    </Typography>
+                </SubHeader>
+            </HeaderSection>
+            <PageBody>
+                <JourneySectionRight
+                    iconImg='Journey1'
+					title="Powering your Profile"
+					subtitle="Shape your stunning professional profile and shine bright like a supernova💫. Showcase your unique talents and interests for the galaxy to see."
+                    buttonTitle="Create Your Profile"
+                    buttonPath='create'
+                    clickFunction={CreatePage}
+                />
+                <JourneySectionLeft
+                    iconImg='Journey2'
+					title="Create with Templates"
+					subtitle="Fire up your engines and learn to create games and apps using our stellar templates."
+                    buttonTitle="Discover Templates"
+                />
+                <JourneySectionRight
+                    iconImg='Journey3'
+					title="Dream with IdeaSpace"
+					subtitle="Dive into our project ideation portal, IdeaSpace, and blast us away with your incredible ideas. Dream bigger than the cosmos!"
+                    buttonTitle="Explore Ideaspace"
+                    clickFunction={IdeaSpacePage}
+                />
+                <JourneySectionLeft
+                    iconImg='Journey4'
+					title="Learn Code and Design"
+					subtitle="Keep growing and thriving with our wealth of resource tools for code and design. Our devotion to continuous learning fuels your journey across the development cosmos."
+                    buttonTitle="Start Learning Now"
+                    clickFunction={LearnPage}
+                />
+                <JourneySectionRight
+                    iconImg='Journey5'
+					title="Join Forces"
+					subtitle="Collaborate and gain real-world experience to reach Core Member status. Become a part of a platform product team team and defy gravitational limitations!🚀"
+                    buttonTitle="Become A Core Member"
+                    clickFunction={JoinPage}
+                />
+            </PageBody>
+            <Footer>
+                <Icons>
+                    <img src={insta} style={{padding:"0px 10px"}}/>
```

Comment:
Move these styles to a class, in SytledLaunchYourJourney.js. It's better to use classes when a style is going to be reused multiple times. `padding: "0px 10px"` is reused, and so is `colour: "white"`.